photo-app/Gemfile

```diff
@@ -34,6 +35,7 @@ gem 'bootsnap', '>= 1.4.4', require: false
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'dotenv-rails'
```

Comment:
Why you need this in test?

Reply:
i was thinking if i test my app then i need the gem to read my `.env` files

Reply:
When you write test, you need to think of it in terms of it being run in an automated way. means, whatever automated way will already inject the env variables for you. Just like in production.

This gem is only required in development